conans/cli/commands/profile.py

```diff
@@ -0,0 +1,131 @@
+from conans.cli.command import conan_command, conan_subcommand, Extender, COMMAND_GROUPS
+from conans.cli.commands import json_formatter
+from conans.cli.output import ConanOutput
+
+
+def profiles_cli_output(profiles):
+    host, build = profiles
+    output = ConanOutput()
```

Comment:
Shouldn't this be `cli_out_write` ? I think `ConanOutput` is writting to the stderr, not stdout.

Reply:
Yes, you are right, lets fix it.